scripts/export-ics.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if [ $# -eq 0 ]
+  then
+    echo "Usage: export-ics.sh email"
+    exit 1
+fi
+
+export AWS_REGION=eu-west-2
+export ENVIRONMENT_NAME=build
```

Comment:
would provide ENVIRONMENT_NAME as a 2nd argument that can use to overrride the env